api_mt/index.md

```diff
@@ -0,0 +1,137 @@
+
+  - [2.4.1 How to obtain an API key](#241-how-to-obtain-an-api-key)
+
+  - [2.4.2 How to set up the Trados Studio 2021™ plugin for Google Cloud Translation Basic (or Google Translate API v2)](#242-how-to-set-up-the-trados-studio-2021-plugin-for-google-cloud-translation-basic-or-google-translate-api-v2)
+
+  - [2.4.3 How to set up the Google API Validator plugin for Trados Studio 2021™ (optional)](#243-how-to-set-up-the-google-api-validator-plugin-for-trados-studio-2021-optional)
+
+[3. How to set up Memoq Translator Pro 9.8™ plugins](#3-how-to-set-up-memoq-translator-pro-98-plugins)  
+
+* [3.1 Amazon Translate (AWS)](#31-amazon-translate-aws)
+
+  -  [3.1.1 How to obtain an API key](#311-how-to-obtain-an-api-key)
+
+  - [3.1.2 How to set up the Memoq Translator Pro 9.8™ plugin for Amazon Translate (AWS) (for a local profile)](#312-how-to-set-up-the-memoq-translator-pro-98-plugin-for-amazon-translate-aws-for-a-local-profile)
+
+* [3.2 DeepL](#32-deepl)
+
+   - [3.2.1 How to obtain an API key](#321-how-to-obtain-an-api-key)
+
+  - [3.2.2 How to set up the Memoq Translator Pro 9.8™ plugin for DeepL](#322-how-to-set-up-the-memoq-translator-pro-98-plugin-for-deepl)
+
+* [3.3 ModernMT](#33-modernmt)
+
+  - [3.3.1 How to obtain an API key](#331-how-to-obtain-an-api-key)
+
+  - [3.3.2 How to set up the Memoq Translator Pro 9.8™ plugin for ModernMT](#332-how-to-set-up-the-memoq-translator-pro-98-plugin-for-modernmt)
+
+* [3.4 Google Cloud Translation Basic (or Google Translate API v2)](#34-google-cloud-translation-basic-or-google-translate-api-v2)
+
+  - [3.4.1 How to obtain an API key](#341-how-to-obtain-an-api-key)
+
+  - [3.4.2 How to set up the Memoq Translator Pro 9.8™ plugin for Google Cloud Translation Basic (or Google Translate API v2)](#342-how-to-set-up-the-memoq-translator-pro-98-plugin-for-google-cloud-translation-basic-or-google-translate-api-v2)
+
+[4. How to obtain API keys for MT plugins](#4-how-to-obtain-api-keys-for-mt-plugins)  
+
+* [4.1 Amazon Translate (AWS)](#41-amazon-translate-aws)
+
+* [4.2 DeepL](#42-deepl)
+
+* [4.3 ModernMT](#43-modernmt)
+
+* [4.4 Google Cloud Translation Basic (or Google Translate API v2)](#44-google-cloud-translation-basic-or-google-translate-api-v2)
+
+[5. References](#5-references)
+## 1. INTRODUCTION
+Machine Translation (MT) technology allows you to automatically translate your text from one language to another. Both Trados Studio 2021™ and memoQ translator pro 9.8™ use plugins to connect to machine translation engine vendors. This manual provides information as to how to obtain API keys from MT vendors and enter them into the plugins. 
+
+*Disclaimer*  
+
+Information contained herein is valid as of October 1st, 2021. Due to frequent changes in user interfaces utilized by MT vendors this information may become obsolete after that date without advance notice. Revision 1.0.
+### 1.1 MT vendors described
+This manual shows how to obtain APIs to the following MTs:
```

Comment:
This guide shows how to API keys for the following machine translation systems:

Reply:
I have introduced this change.